deployment/terraform/assets/cluster.tf

```diff
@@ -281,7 +281,7 @@ resource "aws_rds_cluster_instance" "cluster_instances" {
 }
 
 resource "aws_rds_cluster" "db_cluster" {
-  count               = var.app_instance_count > 0 ? 1 : 0
+  count               = var.db_instance_count > 0 ? 1 : 0
```

Comment:
Same here. This wasn't a typo. The idea was that we only create the DB cluster if we are also deploying the app servers.

Reply:
I think now is working as expected.

Reply:
Perfect :+1: